package.json

```diff
@@ -22,22 +21,13 @@
   },
   "homepage": "https://github.com/flbulgarelli/headbreaker#readme",
   "devDependencies": {
-    "@types/mocha": "^10.0.1",
-    "clean-webpack-plugin": "^1.0.1",
-    "html-webpack-plugin": "^5.5.0",
-    "jsdoc-import-support": "^3.6.4",
-    "mocha": "^10.0.0",
-    "ts-loader": "^9.4.4",
-    "ts-mocha": "^10.0.0",
-    "tsd-jsdoc": "^2.5.0",
-    "typescript": "^5.1.6",
-    "webpack": "^5.0.0",
-    "webpack-cli": "^3.2.1",
-    "webpack-dev-server": "^4.11.1",
-    "webpack-merge": "^4.2.1"
+    "@types/node": "^22.5.4",
+    "typescript": "^5.5.3",
+    "vite": "^5.4.1",
+    "vitest": "^2.1.0"
   },
   "optionalDependencies": {
-    "konva": "^6.0"
-  },
-  "dependencies": {}
+    "canvas": "^2.11.2",
+    "konva": "^9.3.15"
```

Comment:
Is moving to the latest konva version strictly necessary? Perhaps that will imply too much changes at the same time. I think we should focus on moving to `ts`  and then upgrading konva, which might be challenging in its own right.

Reply:
Yeah sure